### PR TITLE
PowerTools does not exist in RHEL8 https://access.redhat.com/discussions/5417621

### PR DESCRIPTION
**Pull request by lessfoobar**
Everything is described in this issue: https://access.redhat.com/discussions/5417621
the PowerTools does not exist in RHEL8

### CODE REVIEW

File: roles/katello_repositories/tasks/main.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 - name: "Set up {{ katello_repositories_environment }} repositories"
   include_tasks: "{{ katello_repositories_environment }}_repos.yml"
 
-- name: enable powertools for libdb_cxx used by qpid-cpp-server-linearstore
+- name: enable codeready-builder for libdb_cxx used by qpid-cpp-server-linearstore
```
Comment (evgeni) [inline roles/katello_repositories/tasks/main.yml:5]:
```suggestion
- name: enable powertools for libdb_cxx used by qpid-cpp-server-linearstore
```

:)

Comment (lessfoobar) [reply roles/katello_repositories/tasks/main.yml:5]: so it should be powertools and not codeready-builder?

Comment (evgeni) [reply roles/katello_repositories/tasks/main.yml:5]: well, for non-Red Hat (so CentOS, Alma, Rocky etc), it's powertools

for Red Hat it's codeready, as in line 9 and further :)

Comment (lessfoobar) [reply roles/katello_repositories/tasks/main.yml:5]: ohhh I see...

**Comment by evgeni** [comment]
LGTM

@ehelms what do you think?

**Review by ehelms — APPROVED** [review]

**Pull request merged**